Advanced-Recruit-Training/src/BatteryStateOfChargeService.cpp

```diff
@@ -7,12 +7,15 @@
 namespace
 {
     const double BATTERY_AMP_HOUR_CAPACITY = 123.0;
+    const double MS_TO_HOURS = 3600000.0;
+    const double S_TO_HOURS = 3600.0;
+    const double HOUR_TO_M = 60.0;
 }
 
 BatteryStateOfChargeService::BatteryStateOfChargeService(double initialStateOfChargePercent)
 : initialStateOfChargePercent_(initialStateOfChargePercent)
 {
-    initalAmpHour_= BATTERY_AMP_HOUR_CAPACITY*initialStateOfChargePercent_/100;
+    initalAmpHour_=BATTERY_AMP_HOUR_CAPACITY-BATTERY_AMP_HOUR_CAPACITY*initialStateOfChargePercent_/100;
```

Comment:
When using multiple math operators, it is nice to have a space between variables, so it is clearer to read, also, use brackets to be clear about what the precedence of your operations is. Finally, this still doesn't produce the initial amp hours correctly